plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/config/LibraryFactory.java

```diff
@@ -151,6 +159,19 @@ private MavenCoordinates getMavenCoordinates(IConfigurationElement[] children) {
     return filters;
   }
 
+  private List<String> getLibraryDependencies(IConfigurationElement[] children) {
+    List<String> libraryDependencies = new ArrayList<>();
+    for (IConfigurationElement libraryDependencyElement : children) {
+      if (libraryDependencyElement.getName().equals(ELMT_LIBRARY_DEPENDENCY)) {
```

Comment:
Do we need to be this much conservative to check the name again, because on line 76, we get these `children` by `getChildren(ELMT_LIBRARY_DEPENDENCY)`, i.e., `children` are already all `ELMT_LIBRARY_DEPENDENCY`?